scripts/deploy.sh

```
@@ -1,96 +1,45 @@
-  IDLE_PORT=8081
+set -e
+
+APP_DIR=/home/ec2-user/app
+JAR_PATH=$(ls $APP_DIR/*.jar | grep -v plain | head -1)
+echo "> JAR 파일: $JAR_PATH"
+
+echo "> 실행 중인 애플리케이션 종료"
+CURRENT_PID=$(pgrep -f '\.jar' || true)
+if [ -n "$CURRENT_PID" ]; then
+  echo "> 종료: $CURRENT_PID"
+  kill -15 $CURRENT_PID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)